entry_types/scrolled/package/src/contentElements/inlineBeforeAfter/BeforeAfter.js

```diff
@@ -1,99 +1,107 @@
-      return;
+  let opts = {};
+  // Transform slider-related props into the format that
+  // react-compare-image needs
+  if (!slider) {
+    opts = {...opts, sliderLineWidth: 0, handle: <React.Fragment/>};
+  }
+  if (slider) {
+    if (!slider_handle) {
+      opts = {...opts, handle: <React.Fragment/>};
     }
-
-    setTimeout(handler, 0);
-
-    if (state === 'active') {
-      window.addEventListener('scroll', handler);
-
-      return function() {
-        window.removeEventListener('scroll', handler);
-      }
+    if (slider_color) {
+      opts = {...opts, sliderLineColor: slider_color};
     }
-  }, [current, setBeforeAfterPos, scrollPos, state, setIsSliding]);
+  }
 
-  const awsBucket = '//s3-eu-west-1.amazonaws.com/de.codevise.pageflow.development/pageflow-next/presentation-images/';
-  const beforeImage = awsBucket+'before_after/haldern_church1.jpg';
-  const afterImage = awsBucket+'before_after/haldern_church2.jpg';
+  if (current) {
+    // Size labels according to initial slider position. Unit is %,
+    // left and right spacing is 5%, so we subtract 10. 90 = 100 - 10.
+    const beforeLabelWidth = initial_slider_position - 10;
+    const afterLabelWidth = 90 - initial_slider_position;
+    // Compute initial slider coordinate and pass it as a CSS
+    // variable, so that before/after images can wiggle together with
+    // the slider
+    const initialSliderPos = initial_slider_position / 100;
+    const containerWidth = current.getBoundingClientRect().width;
+    const initialRectWidth = initialSliderPos * containerWidth;
+    current.style.setProperty('--before-label-max-width', beforeLabelWidth + '%');
+    current.style.setProperty('--after-label-max-width', afterLabelWidth + '%');
+    current.style.setProperty('--initial-rect-width', initialRectWidth + 'px');
+  }
 
   return (
-    <div ref={beforeAfterRef} className={styles.container}>
-      <ReactCompareImage leftImage={beforeImage} rightImage={afterImage}
-                         sliderPosition={beforeAfterPos} setSliderPosition={setBeforeAfterPos}
-                         isSliding={isSliding} setIsSliding={setIsSliding}
-                         leftImageLabel={leftImageLabel} rightImageLabel={rightImageLabel}
-                         classicMode={slideClassic}
-                         wiggle={wiggle} />
+    <div ref={beforeAfterRef}
+      className={cx({[styles.selected]: isSelected, [styles.wiggle]: wiggle},
+      styles.container)}>
+      <InitialSliderPositionIndicator parentSelected={isSelected}
+        position={initial_slider_position}/>
+      {/* onSliderPositionChange: Prevent wiggle if user uses slider */}
+      <ReactCompareImage leftImage={beforeImageUrl} rightImage={afterImageUrl}
+                         leftImageLabel={before_label} rightImageLabel={after_label}
+                         leftImageAlt={beforeImageAlt} rightImageAlt={afterImageAlt}
+                         sliderPositionPercentage={initial_slider_position/100}
```

Comment:
Since you're already calculating it up top, you could use `initialSliderPos` here.